web/admin/src/views/ProfileAccountPage.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+import { onBeforeMount } from 'vue'
+import { useAuthUserStore } from '@/stores/authUser'
+import { useRouter } from 'vue-router'
+import ProfileTabs from '@/components/ProfileTabs.vue'
+import { getAuth, sendEmailVerification } from 'firebase/auth'
+
+const props = defineProps({
+	uid: String
+})
+
+const user = useAuthUserStore()
+const router = useRouter()
+const auth = getAuth()
+
+const sendVerificationEmail = () => {
+	sendEmailVerification(auth.currentUser).then(() => {
+		window.alert(`Verification email sent to ${user.data.email}`)
+	})
+}
+
+onBeforeMount(async () => {
+	if (props.uid === '' || props.uid === undefined) {
+		props.uid = user.data.uid
+	}
+})
+</script>
+
+<template>
+	<div class="container">
```

Comment:
Let's change this to a component, and drop the `class="container"`

Reply:
Should I remove the account tab?

Reply:
Yes, put the component above the registration form, something like this:

![image](https://github.com/ncosd/food-pantry-app/assets/136589/06bbc972-6cda-4a77-9b8c-dba28cf5f088)